tests/integ/modin/test_sql_counter.py

```diff
@@ -80,22 +85,30 @@ def test_sql_counter_with_context_manager_inside_loop():
 
 @sql_count_checker(no_check=True)
 def test_sql_counter_with_multiple_checks():
-    with SqlCounter(query_count=1, describe_count=1):
+    with SqlCounter(
+        query_count=1, describe_count=1 if is_sql_simplifier_enabled() else 0
+    ):
         df = pd.DataFrame({"a": [1, 2, 3]})
         assert len(df) == 3
 
-    with SqlCounter(query_count=1, describe_count=1):
```

Comment:
and the describe call seems currently only work for SqlCounter, not the annotation, I created a ticket here https://snowflakecomputing.atlassian.net/browse/SNOW-1704919